src/xc/lda_x.jl

```diff
@@ -1,4 +1,10 @@
 """
 LDA Slater exchange (DOI: 10.1017/S0305004100016108 and 10.1007/BF01340281)
 """
-energy_per_particle(::Val{:lda_x}, ρ) = -3/4 * cbrt(3/π * ρ)
+function energy_per_particle(::Val{:lda_x}, ρ) 
+    #@warn "lda_x energy per particle"
+    if ρ <= 1e-8
+        return 0.0
+    end
```

Comment:
Same thing as before. Actually we should probably put this one level up (to where these functions are called) anyway.